src/mip/bound_strengthening.jl

```diff
@@ -0,0 +1,66 @@
+function bound_strengthening!(ps::PresolveData{T}, j::Int, ϵ::T=eps(T), ϵ_int::T=eps(T)) where{T}
+    # Column was already removed or the variable is continous.
+    ps.colflag[j] || return nothing
+    (ps.var_types[j] == CONTINUOUS) && return nothing
```

Comment:
There's a variant of this that works just fine for continuous variables: you just get rid of the floors and ceilings, essentially. Would be good to support that in this PR if it isn't too much extra code.

Reply:
Is it implemented somewhere in the LP as shown in the MIP presolve reductions roadmap?

Reply:
It does say that in the roadmap, but I'm not sure where it is in the library. @mtanneau?

Reply:
I'm not 100% sure how continuous variables should be handled.
For MIP problems, I'd tend to say that we want to have tighter bounds, since it can only improve the LP relaxation.

For continuous problems, it's actually better to _drop_ a bound.
Namely, if a variable bound `x >= ℓ` can be strengthened to `x >= ℓ'`, then the original bound is never active and `x` can be treated as a free variable (these are called "implied-free" variables). For simplex, implied-free variables can be put in the basis and will never leave it. Some IPM solvers don't like free variables though.
Nevertheless, one should keep track of variable bounds because they help tighten other constraints. This would likely come with some book-keeping for dual postsolve, too.

I don't think I implemented it anywhere, partly for the above reason.

Reply:
For now, we can:
* Skip continuous variables altogether
* Strengthen bounds on continuous variables, but only if the problem is a MIP (because then we don't care about dual postsolve)

Reply:
Do we want to implement strengthening bounds on continuous variables in a new file in LP folder? We can turn on this routine only when we want.

Reply:
I would think we leave a single implementation in the mip/ directory as you have here, but apply it to each variable regardless of its variable type. So, essentially, remove this line.